package.json

```diff
@@ -27,15 +28,18 @@
     "axios": "^0.18.0",
     "body-parser": "^1.18.3",
     "cassandra-driver": "^3.5.0",
+    "chai": "^4.1.2",
```

Comment:
Are you testing with mocha, chai, and stambul? If not we can get rid of these before deployment.